### PR TITLE
Wat 380 record command functional

### PR DESCRIPTION
**Pull request by shenterova**
Create run command implementation:

Should init same modules, except test finder
Should start server with recorder app
Should listen to server event with new content
Should pass new test to run controller
Should not fail process if there is error, log this error and return it to server for displaying to user

### CODE REVIEW

File: packages/testring-child-process/test/fork.spec.ts [hunk 1]
```diff
@@ -7,21 +7,21 @@ import { fork } from '../src/fork';
...
-            } else {
-                chai.expect(ps['spawnfile']).to.include('ts-node');
-
-                callback();
-            }
-        });
-
-        ps.kill();
-    });
+    // it('should fork .ts files with ts-node', (callback) => {
```
Comment (johnthecat) [inline packages/testring-child-process/test/fork.spec.ts]: remove this test plz

File: packages/testring-child-process/test/resolve-binary.spec.ts [hunk 2]
```diff
@@ -1,36 +0,0 @@
-/// <reference types="mocha" />
```
Comment (johnthecat) [inline packages/testring-child-process/test/resolve-binary.spec.ts]: restore file, please

File: packages/testring-cli/src/index.ts [hunk 3]
```diff
@@ -79,23 +80,25 @@ export const runCLI = (argv: Array<string>) => {
     const args = yargs.parse(argv);
     const command = args._[2];
 
-    let commandExecution;
-
     switch (command) {
         case 'run':
-            commandExecution = runTests(argv, process.stdout);
+            runTests(argv, process.stdout).catch((exception) => {
```
Comment (johnthecat) [inline packages/testring-cli/src/index.ts]: Why did you move `catch(..)` back to command call? I think solution from master was better

File: packages/testring-test-run-controller/src/test-run-controller.ts [hunk 4]
```diff
@@ -37,8 +38,19 @@ export class TestRunController extends PluggableModule implements ITestRunContro
...
 
+        if (Array.isArray(this.testQueue)) {
+            this.testQueue.push(...testQueue);
+        } else {
+            this.testQueue = testQueue;
+        }
+
         loggerClientLocal.debug('Run controller: tests queue created.');
 
+        return this.executeQueue(this.testQueue);
```
Comment (johnthecat) [inline packages/testring-test-run-controller/src/test-run-controller.ts]: move `executeQueue` right after `this.testQueue = testQueue;`

**Comment by shenterova** [comment]
comments applied
